### PR TITLE
Enhancement/no shape lists

### PR DESCRIPTION
**Pull request by jb-roa**

### CODE REVIEW

File: src/app/examples/list/no-shape/list-no-shape-example.component.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
+@import "../../../../kirby/scss/utils";
+
+kirby-card {
+  margin: 5px;
```
Comment (jakobe) [inline src/app/examples/list/no-shape/list-no-shape-example.component.scss]: Consider extracting shadow units (offset-x | offset-y | blur-radius | spread-radius) from `src\kirby\scss\base\_functions.scss` into separate scss variables and using them instead.

File: src/app/examples/list/no-shape/list-no-shape-example.component.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  width: 330px;
```
Comment (jakobe) [inline src/app/examples/list/no-shape/list-no-shape-example.component.scss]: Consider calculating from `$breakpoints` map in `src\kirby\scss\base\_variables.scss`

File: src/app/examples/list/no-shape/list-no-shape-example.component.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<h2>List with no background shape:</h2>
+<h3>Removes background, border, shadow and padding from the list and the list item containers to provide flexibility
+</h3>
+<kirby-list [items]="items" shape="none" hasItemSpacing="true">
+
+  <kirby-card *kirbyListItem="let item">
+    <kirby-card-header *ngIf="item.amount < 0" title="Account is overdraft"></kirby-card-header>
+    <kirby-list-flex-item hasPadding="true">
```
Comment (jakobe) [inline src/app/examples/list/no-shape/list-no-shape-example.component.html]: `hasPadding` property doesn't exist on `kirby-list-flex-item` - move up to `kirby-card` element

File: src/app/examples/list/no-shape/list-no-shape-example.component.scss [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+  @include media(">=large") {
+      width: 500px;
+  }
+
+  kirby-card-header {
+      padding: size('xxxs');
```
Comment (jonasjensen-brolstark) [inline src/app/examples/list/no-shape/list-no-shape-example.component.scss]: Nitpicking: You have 4 spaces per tap here but 2 spaces other places in the file.
And all the other code uses 2 spaces. 😃

File: src/kirby/components/list/list.component.ts [hunk 5]
```diff
@@ -24,7 +24,9 @@ import { GroupByPipe } from './pipes/group-by.pipe';
 import { ListSwipeAction } from './list-swipe-action';
 import { ThemeColor } from '@kirbydesign/designsystem/helpers/theme-color.type';
 
-export type ListShape = 'square' | 'rounded';
+export type ListShape = 'square' | 'rounded' | 'none';
+
+export type ItemSpacing = 'true' | 'false';
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/list/list.component.ts]: What is the purpose of making `ItemSpacing` a string based type over a boolean?

Comment (jb-roa) [reply src/kirby/components/list/list.component.ts]: Well.. 
1: hasItemSpacing="false" is resulting in a string value, i.e.: 'false' is truthy.
And 2: We hope to extend this later, with values such as "compact" or "spacey"

File: src/app/examples/list/no-shape/list-no-shape-example.component.scss (hunk 4 above)
Comment (jb-roa) [reply src/app/examples/list/no-shape/list-no-shape-example.component.scss]: Doh.. my VS Code stopped auto formatting for some reason. Will fix :-)

File: src/app/examples/list/no-shape/list-no-shape-example.component.scss [hunk 6]
```diff
@@ -0,0 +1,34 @@
+@import "../../../../kirby/scss/utils";
+
+kirby-card {
+  margin: map-get($shadow-offset-y, 2);
```
Comment (jb-roa) [inline src/app/examples/list/no-shape/list-no-shape-example.component.scss]: @jakobe I'm using the variables as requested now. Luckily, the shadow-offset can be used here, but in reality the margin should be based on a calculation I guess, depending on what side the margin is on.
However I would like this not to be in scope of this PR, as I don't see it as a trivial thing to implement.

File: src/kirby/scss/base/_variables.scss [hunk 7]
```diff
@@ -63,3 +63,29 @@ $breakpoints: (
   xlarge: 60em, 
   xxlarge: 80em 
 ) !default;
+
+/* Shadow properties
```
Comment (jb-roa) [inline src/kirby/scss/base/_variables.scss]: @jakobe Please le me know if this is what you were looking for.

File: src/app/examples/list/no-shape/list-no-shape-example.component.scss (hunk 4 above)
Comment (jb-roa) [reply src/app/examples/list/no-shape/list-no-shape-example.component.scss]: Fixed manually. My Prettier is still broken for some reason.

File: src/app/examples/list/no-shape/list-no-shape-example.component.html (hunk 3 above)
Comment (jb-roa) [reply src/app/examples/list/no-shape/list-no-shape-example.component.html]: Removed

File: src/app/examples/list/no-shape/list-no-shape-example.component.scss (hunk 2 above)
Comment (jb-roa) [reply src/app/examples/list/no-shape/list-no-shape-example.component.scss]: Now using map-get($breakpoints, ...) instead. Is this what you meant?

File: src/app/examples/list/no-shape/list-no-shape-example.component.scss (hunk 1 above)
Comment (jb-roa) [reply src/app/examples/list/no-shape/list-no-shape-example.component.scss]: Extracted shadow variables. However calculating the margin based on them is actually not that trivial after all as I see it. Something like:
top = (spread - v_shadow + 0.5*blur)
right = (spread + h_shadow + 0.5*blur)
bottom = (spread + v_shadow + 0.5*blur)
left = (spread - h_shadow + 0.5*blur)

I would prefer it not to be in scope of this PR.

File: src/kirby/components/list/list.component.ts (hunk 5 above)
Comment (jonasjensen-brolstark) [reply src/kirby/components/list/list.component.ts]: 1. I actually didn't know. Well today i learned.
2. Fine.

Comment (jakobe) [reply src/kirby/components/list/list.component.ts]: We don't do this in any other components, so I'm reverting to boolean and closing this conversation

**Comment by jb-roa** [comment]
Fixes #504

**Pull request merged**